app/src/main/java/com/example/easycloset/ClosetFragment.java

```diff
@@ -0,0 +1,30 @@
+package com.example.easycloset;
+
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+
+public class ClosetFragment extends Fragment {
+
+    public ClosetFragment() {
+        // Required empty public constructor
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
```

Comment:
you don't have to override onCreate if there is no implementation

Reply:
fixed